### DISCUSSION TITLE
Script Error when displaying the Pnp Search Box

### DISCUSSION THREAD

**Discussion by Phentrin** [microsoft-search/pnp-modern-search#2812] [Q&A]
CDN issue to download js file for standard user.

**Describe the bug**
ERREUR :
Script error for: 544c1372-7e5a-49ec-8db6-812f76c375f2_4.8.0
http://requirejs.org/docs/errors.html#scripterror

Error: Script error for: 544c1372-7e5a-49ec-8db6-812f76c375f2_4.8.0
http://requirejs.org/docs/errors.html#scripterror
    at B (eval at e (https://res-1.cdn.office.net/files/sp-client/sp-pages-assembly_fr-fr_2bff8c924116e7428f5cb588c19824b3.js:70:870977), <anonymous>:8:252)
    at HTMLScriptElement.onScriptError (eval at e (https://res-1.cdn.office.net/files/sp-client/sp-pages-assembly_fr-fr_2bff8c924116e7428f5cb588c19824b3.js:70:870977), <anonymous>:30:244)

**To Reproduce**
Detailed steps to reproduce the behavior:
1. Upload the pnp-modern-search-parts-v4.sppkg to the Global App Catalog
2. Deploy to All Sites
3. A the Web Part on a Modern Page & Publish
4. Connect as a standard user to display the page
5. See error

**Expected behavior**
Display the web part

**Screenshots**
The solution is deployed in Global App Catalog
![image](https://user-images.githubusercontent.com/12797998/221880492-534b2e77-96ea-406a-9b81-591b460e8f08.png)

CDN error with a standard user
![image](https://user-images.githubusercontent.com/12797998/221879043-05e08a61-0959-4c74-9cff-27b9e88ef810.png)
![image](https://user-images.githubusercontent.com/12797998/221879543-ecc15857-3893-4d30-97df-8d6bcfc9bd97.png)

It works with the Administrator account
![image](https://user-images.githubusercontent.com/12797998/221880108-76b62e8a-af24-4016-8b03-443ce3a23627.png)
![image](https://user-images.githubusercontent.com/12797998/221880240-c53fc597-3e51-46ca-9a89-1b53d93aecf1.png)

**Comment by wobba** [comment]
Common error and SPFx related. You can try to re-upload the web part, re-add to sites. Did you upgrade from a previous version?

**Reply by jonasesser** [reply]
We have the same issue. But on a fresh install. Any further hints to resolve the issue?

Something went wrong
If the problem persists, contact the site administrator and give them the information in Technical Details.
TECHNICAL DETAILS
ERROR:
Script error for: 544c1372-7e5a-49ec-8db6-812f76c375f2_4.8.0/SearchBoxWebPartStrings
http://requirejs.org/docs/errors.html#scripterror

CALL STACK:
Error: Script error for: 544c1372-7e5a-49ec-8db6-812f76c375f2_4.8.0/SearchBoxWebPartStrings
http://requirejs.org/docs/errors.html#scripterror
    at B (eval at e (https://res-1.cdn.office.net/files/sp-client/sp-pages-assembly_en-us_4fd1adbdbc45c9ba163860e9fcb8b44f.js:1:1), <anonymous>:8:252)
    at HTMLScriptElement.onScriptError (eval at e (https://res-1.cdn.office.net/files/sp-client/sp-pages-assembly_en-us_4fd1adbdbc45c9ba163860e9fcb8b44f.js:1:1), <anonymous>:30:244)

**Reply by jonasesser** [reply]
I think the package was wrongly build for dev instead for prod. Not find the correct builds command an website :D any hints?

**Reply by jonasesser** [reply]
npm run build or npm run serve?

**Reply by jonasesser** [reply]
![image](https://github.com/microsoft-search/pnp-modern-search/assets/19924406/105e510a-0c00-4382-a017-c89e6349d1b8)

**Reply by jonasesser** [reply]
will try it with gulp package-solution --production

**Reply by athul-or** [reply]
Hi @jonasesser ,

Did you find any solution? I also have this issue.

**Reply by wobba** [reply]
Our packages for download are not dev built. You should be able to download the sppkg and upload it. If you have CDN.. then wait a bit and clear all browser cache when re-trying.

**Reply by athul-or** [reply]
Hi @wobba ,
I tried clearing cache. For my administrator the pnp webparts are showing after a hard refresh and after a normal refresh the same error.But for both users its working in Edge. In Chrome console its showing the error failed to execute script because strict MIME type checking is enabled.

**Reply by wobba** [reply]
Well.. not sure how to tackle this. Seems a SPFx issue, not a web part issue as loading is done by the framework.  Are you using CDN or not? Are permissions set correctly?
